pyaedt/emit_core/results/revision.py

```diff
@@ -19,6 +18,10 @@ class Revision:
         Name of the revision to create. The default is ``None``, in which case a
         default name is given.
```

Comment:
Should this have the same second sentence as in the previous file?

The default is ``None``, in which case the name of the current design revision is used.